Carmu/Sources/Views/SessionStart/SessionStartNoCrewView.swift

```diff
@@ -107,22 +110,24 @@ final class NoCrewFrontView: UIView {
 
         return label
     }()
+
     lazy var createCrewButton: UIButton = {
         let button = UIButton()
         button.setTitle("크루 만들기", for: .normal)
         button.setTitleColor(UIColor.semantic.textSecondary, for: .normal)
-        button.titleLabel?.font = UIFont.carmuFont.headline1
+        button.titleLabel?.font = UIFont.carmuFont.subhead3
         button.backgroundColor = UIColor.semantic.accPrimary
-        button.layer.cornerRadius = 30
+        button.layer.cornerRadius = deviceHeight == 667 ? 15 : 30
```

Comment:
혹시 모르니 deviceHeight와 비교하는 조건들을 `==`보다 `<=`로 적어주면 어떨까 싶어요
func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath)에 적어주신 것처럼요!

```suggestion
        button.layer.cornerRadius = deviceHeight <= 667 ? 15 : 30
```